Automattic/wp-calypso
# Media: Remove flux media actions

The media flux actions are no longer necessary. We're removing them here, along with some related cleanups. Related to #46742 where we're removing the media flux store.

#### Changes proposed in this Pull Request

* Mirror media query clearing in the media modal with Redux when doing it with the flux action.
* Remove an unused `copyExternal` method from the media modal.
* Make imports relative to Calypso root in the media modal.
* Remove `MediaActions` usage from MediaListData.
* Remove `MediaActions` usage from media modal.
* Remove unused legacy flux media actions along with their tests.

#### Testing instructions

* Test `/media/:site` and verify it works well like before (uploading, editing, filtering, searching, adding images from external sources).
* Verify image uploading, editing, and adding images from an external source such as Google Photos or Pexels in the media modal still works well.

#### Note

We're not removing the flux media actions from the media lib README because we're removing that file completely in #46742.
